modules/authorization/role-assignments/subscription/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Role Assignment on Subscription level",
+  "summary": "With this module you can perform role assignments on a subscription level",
```

Comment:
Should the README.md be updated to "This module deploys Role Assignments at a Subscription level." and then this file updated accordingly?